server/src/main/java/org/apache/druid/server/log/DefaultRequestLogEvent.java

```diff
@@ -55,23 +55,26 @@ public final class DefaultRequestLogEvent implements RequestLogEvent
    */
   @JsonValue(value = false)
   @Override
-  public Map<String, Object> toMap()
+  public EventMap toMap()
   {
-    final Map<String, Object> map = new HashMap<>();
-    map.put("feed", getFeed());
-    map.put("timestamp", getCreatedTime());
-    map.put("service", getService());
-    map.put("host", getHost());
-    if (getQuery() != null) {
-      map.put("query", getQuery());
-    }
+    final EventMap.Builder builder = EventMap
+        .builder()
+        .put("feed", getFeed())
+        .put("timestamp", getCreatedTime())
+        .put("service", getService())
+        .put("host", getHost())
+        .putNonNull("query", getQuery());
+
     if (getSql() != null) {
-      map.put("sql", getSql());
-      map.put("sqlQueryContext", getSqlQueryContext());
+      builder.put("sql", getSql())
+             .put("sqlQueryContext", getSqlQueryContext());
     }
-    map.put("remoteAddr", getRemoteAddr());
-    map.put("queryStats", getQueryStats());
-    return map;
+
+    builder
+        .put("remoteAddr", getRemoteAddr())
```

Comment:
the indentation seems a bit weird. The first put can be on the same line

Reply:
Right, I'll fix that